bitrise/setup.go

```diff
@@ -77,12 +80,17 @@ func RunSetup(appVersion string, setupMode SetupMode, doCleanSetup bool) error {
 		return fmt.Errorf("Failed to do common/platform independent setup, error: %s", err)
 	}
 
+	deps := []string{}
```

Comment:
This is empty, are you sure it's needed?

Reply:
Not needed right now, added this to install zstd. Should I restore all setup.go cahnges?